automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -335,11 +335,12 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
 
     notebookPage.executeCell("from hail import *") shouldBe None
     notebookPage.executeCell("hc = HailContext(sc)").get should include(welcomeToHail)
-
     notebookPage.executeCell(s"chr20vcf = '${vcfPath.toUri}'") shouldBe None
-    notebookPage.executeCell("imported = hc.import_vcf(chr20vcf)").get should include("Hail: INFO: Coerced almost-sorted dataset")
 
-    notebookPage.executeCell("imported.summarize().report()").get should include(vcfSummary)
+    val fiveMinuteTimeout = 5 * 60
```

Comment:
It's slower with only one preemptible

Reply:
Would be nice to use `FiniteDuration` :)